src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -267,6 +268,51 @@ export function transformProviderFacilities(providerFacilities) {
   }));
 }
 
+/**
+ * Concatenates incident location address object into location string. This will ignore null
+ *  or undefined address fields
+ * @param {Object} incidentLocation location address with city, state, country, and additional details
+ * @returns {String} incident location string
+ */
+export function concatIncidentLocationString(incidentLocation) {
+  return [
+    incidentLocation.city,
+    incidentLocation.state,
+    incidentLocation.country,
+    incidentLocation.additionalDetails,
+  ]
+    .filter(locationField => locationField && locationField.length > 0)
```

Comment:
If `locationField === ''`  then `locationField` is falsey. I don't think you need that second condition.
![image](https://user-images.githubusercontent.com/12970166/50364224-e6760800-0523-11e9-95da-31de1ae2c03f.png)